### ISSUE TITLE
Make the package look more lightweight

### ISSUE THREAD

**Issue by stubailo** [apollographql/apollo-fetch#37]
This is feedback from @tmeasday - the readme makes this package look like it's doing a lot more than just a fetch, when in fact we want it to be a super thin wrapper. Let's update the readme/description to make people feel better about using it.

**Comment by tmeasday** [comment]
Yeah, it's an unusual use case where people are explicitly not looking for features. I normally don't like "just X kb" type talk but it's probably justified in this case.

**Comment by stubailo** [comment]
Yep - we could make the readme just like a single code sample and then link off to more features.
